hasher-matcher-actioner/scripts/script_utils.py

```diff
@@ -62,31 +100,169 @@ def send_single_submission_url(
         additional_fields: t.List[str],
         api_path: str = "/submit/",
     ):
-
         payload = {
             "submission_type": "POST_URL_UPLOAD",
             "content_id": content_id,
             "content_type": "PHOTO",
             "content_bytes_url_or_file_type": "image/jpeg",
             "additional_fields": [],
         }
-
-        payload_bytes = json.dumps(payload).encode()
-
-        response = requests.post(
+        response_json = self.session.post(
             self._get_request_url(api_path),
-            headers=self._get_header(),
-            data=payload_bytes,
-        )
+            data=json.dumps(payload).encode(),
+        ).json()
 
-        response_json = response.json()
         put_response = requests.put(
             response_json["presigned_url"],
             data=file,
             headers={"content-type": "image/jpeg"},
         )
 
+    def get_content_hash_details(
+        self,
+        content_id: str,
+        api_path: str = "/content/hash/",
```

Comment:
Why is the `api_path` an argument? Not seeing the utility here? Do we expect this to change?

Reply:
I found overriding helpful for debugging and testing. If I create `/content/new-hash/` for example that I want to function the same but not replace the old impl just yet.
It's roughly the same number of lines but if these methods become too much like Swiss Army Knife Anti-patterns I'll refactor.